src/main/java/ca/on/oicr/gsi/dimsum/service/filtering/CompletedGate.java

```diff
@@ -107,8 +108,13 @@ public boolean qualifyCase(Case kase) {
 
     @Override
     public boolean qualifyRequisition(Requisition requisition) {
-      return requisition.getInformaticsReviews().stream().allMatch(reqQc -> reqQc.isQcPassed());
-    }    
+      RequisitionQc reqQc = requisition.getInformaticsReviews().stream().max(Comparator.comparing(RequisitionQc::getQcDate)).orElse(null);
+      if (reqQc != null && reqQc.isQcPassed()) {
+        return true;
+      }
+      return false;
```

Comment:
This looks good now, but qualifyCase needs to do the same. Ideally, just modify `Helpers#isCompletedRequisitionQc` to do this and take a `Requisition` instead of a `Case`, then you can use that in all 6 places